docs/dictionary/command/filter.lcdoc

```diff
@@ -71,20 +71,17 @@ Description:
 Use the <filter> command to pick specific lines, items, keys or elements
 in a container or expression.
 
-The <filter>...with form and the <filter>...matching form retain the
+The '<filter>...with' form and the '<filter>...matching' form retain the
 lines, items, keys or elements that contain a match for the specified
 <filterPattern>.
 
-The <filter>...without form and the <filter>...not matching form discard
+The '<filter>...without' form and the '<filter>...not matching' form discard
 the lines, items, keys or elements that do not contain a match for the
 specified <filterPattern>. 
 
 If you don't specify lines, items, keys or elements then lines are
 filtered by default.
 
-If a regex pattern is specified, the <filterPattern> evaluates to a
-regular expression.
-
```

Comment:
@seaniepie in the section below describing the wildcard special characters could you wrap them all in back ticks? The `- *` one in particular is causing the markdown renderer to render an extra indent on the unordered list.